README.md

```diff
@@ -6,12 +6,66 @@ IOpipe Analytics & Distributed Tracing Agent
 
 This package provides analytics and distributed tracing for event-driven applications running on AWS Lambda.
 
-# Installation & usage
+- [Installation](#installation)
+- [Usage](#usage)
+  - [Custom Metrics](#custom-metrics)
+  - [Tags](#tags)
+- [Configuration](#configuration)
+  - [Methods](#methods)
+  - [Options](#options)
 
-Install by requiring this module, passing it an object with your project token ([register for access](https://www.iopipe.com)), and it will automatically monitor and collect metrics from your applications running on AWS Lambda.
+# Installation
+
+Install using your package manager of choice,
+
+`npm install -S @iopipe/core`
```

Comment:
Can drop `-S` flag if you like

Reply:
Thanks, I forgot that that's in LTS node installs now